amundsen_application/static/js/components/common/ResourceListItem/SchemaInfo/index.tsx

```diff
@@ -23,7 +25,7 @@ const SchemaInfo: React.SFC<SchemaInfoProps>  = ({ schema, placement, desc }) =>
      placement={ placement }
      overlay={popoverHoverFocus}
      >
-      <span>{ schema }</span>
+      <span className="underline">{ schema }</span>
```

Comment:
You might not need the span tag anymore, I would check it out.

Reply:
If I do not use span, it given an error on browser. Let me know if it works for you

Reply:
I see, sure this is OK too